sdk/servicebus/service.projects

```diff
@@ -21,6 +21,9 @@
   <ItemGroup Condition="'$(SDKType)' == 'client'">
     <ProjectsToIncludeBySDKType Include="$(MSBuildThisFileDirectory)Azure.Messaging.*\**\*.csproj" />
     <ProjectsToIncludeBySDKType Include="$(MSBuildThisFileDirectory)Microsoft.Azure.WebJobs.Extensions.ServiceBus\**\*.csproj" />
+    <ProjectReference Update="$(MSBuildThisFileDirectory)Azure.Messaging.*\tests\*.csproj">
```

Comment:
Can you help me understand what the effect of this is?

Reply:
This is essentially saying that the Azure.Messaging.ServiceBus tests cannot be run in parallel with other test assemblies.

Reply:
@heaths discovered this recently and added to the docs - https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/core/Azure.Core.TestFramework/README.md#running-live-tests-serially